src/pages/ProductPage.jsx

```diff
@@ -20,9 +20,27 @@ import ReviewsSection from "../components/ReviewsSection/ReviewsSection";
 import Reviews from "../components/Reviews/Reviews";
 import MeetYourHostSection from "../components/MeetYourhostSection/MeetYourHostSection";
 import Amenities from "../components/Amenities/Amenities";
-
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { useParams } from "react-router-dom";
 
 const ProductPage = () => {
+  const [error, setError] = useState(null);
+  const [place, setPlace] = useState(null);
+  const { productId } = useParams();
+  console.log(productId);
+  useEffect(() => {
+    axios
+      .get(`http://localhost:8800/places/${productId}`)
```

Comment:
Better if you could use the BASE_URL constant here because if we decide to change it at an later stage we only need to change it in one place.

Reply:
This part is related to the frontend team, here I just showed them how to access the data!